deprecated/perception/navigator_vision/nodes/pinger_locator.py

```diff
@@ -20,26 +19,29 @@
         if processed_ping.freq > 35000 and processed_ping.freq < 36000:
             freq_dev = abs(target_freq - processed_ping.freq)
             print "Trustworthy pinger heading"
-            hydrophones_enu_p, hydrophones_enu_q = tf.lookupTransform("/hydrophones", "/enu", processed_ping.header.stamp)
+            hydrophones_enu_p, hydrophones_enu_q = tf.lookupTransform(
+                "/hydrophones", "/enu", processed_ping.header.stamp)
             pinger_enu_p = navigator_tools.rosmsg_to_numpy(tf.transformPoint())
             dir_ = navigator_tools.rosmsg_to_numpy(processed_ping.position)
-        	mv_mag = 2
+            mv_mag = 2
             mv_hyd_frame = dir_ / np.linalg.norm(dir_)
-            pinger_move = navigator.move.set_position(navigator_tools.rosmsg_to_numpy(processed_ping.position)).go()
+            pinger_move = navigator.move.set_position(navigator_tools.rosmsg_to_numpy(processed_ping.position)).go()  # noqa
```

Comment:
why noqa on this one?

Reply:
Doesn't seem that pinger_move is used anywhere, so removing the assignment should work